google-kubernetes-engine/README.md

```diff
@@ -0,0 +1,64 @@
+# Google Kubernetes Engine
+
+To monitor Google Kubernetes Engine, integrate SignalFx with [Google Cloud
+Platform](https://github.com/signalfx/integrations/tree/master/gcp)[](sfx_link:gcp).
+
+## Properties
+
+Each metric synced from StackDriver is given a special `gcp_id` dimension that
+uniquely identifies that resource (node, pod, or container) and on which
+certain metadata properties will be set, which will cause those properties to
+be propagated to all MTSs with that `gcp_id` dimension.
+
+### k8s_node
+
+The `gcp_id` dimension is in the format `<project_id>_<cluster_name>_<node_name>`.
+
+The following properties are synced to `gcp_id`:
+
+ - All labels on the node are synced to properties of the form: `gcp_<label_name>: <label_value>`
+ - `gcp_node_uid`: The UID of that node
+
+### k8s_pod
+
+The `gcp_id` dimension is in the format `<project_id>_<cluster_name>_<namespace_name>_<pod_name>`.
+
+The following properties are synced to `gcp_id`:
+
+ - All pod labels are synced to properties of the form: `gcp_<label_name>: <label_value>`
+ - `gcp_pod_uid`: The UID of the pod
+ - `gcp_node_name`: The name of the node on which the pod is running
```

Comment:
@keitwb mostly wondering if you think this dimension is explicit enough as opposed to something like `gcp_k8s_node_name` - or maybe since the nodes are probably just underlying compute nodes, maybe this is fine.

Reply:
Resolved over Slack.